backend/tracim_backend/models/reaction.py

```diff
@@ -0,0 +1,36 @@
+from sqlalchemy import Column
+from sqlalchemy import ForeignKey
+from sqlalchemy import Sequence
+from sqlalchemy import Unicode
+from sqlalchemy import UniqueConstraint
+from sqlalchemy.orm import relationship
+from sqlalchemy.types import Integer
+
+from tracim_backend.models.auth import User
+from tracim_backend.models.data import Content
+from tracim_backend.models.meta import DeclarativeBase
+from tracim_backend.models.mixins import CreationDateMixin
+
+
+class Reaction(CreationDateMixin, DeclarativeBase):
+    __tablename__ = "reaction"
+    __table_args__ = (UniqueConstraint("author_id", "content_id", "value"),)
+    MAX_REACTION_VALUE_LENGTH = 255
+
+    reaction_id = Column(
+        Integer, Sequence("seq__reaction__reaction_id"), autoincrement=True, primary_key=True
+    )
+    content_id = Column(
+        Integer, ForeignKey("content.id", onupdate="CASCADE", ondelete="CASCADE",), nullable=False,
+    )
+    value = Column(Unicode(MAX_REACTION_VALUE_LENGTH), nullable=False)
+    author_id = Column(Integer, ForeignKey("users.user_id"), nullable=False)
```

Comment:
When a user is trashed: Nothing
When a user is deleted (using tracimcli): If the reaction belong to a workspace where the user is not the owner, this will failed.

The `tracimcli` case was not supported until now, so i did adjust it.
We don't have any cascade on user now. We may add them but it also make sense to me as a security rule, for not deleting entity we do not expect to delete.

Reply:
note: it's far from funny to add new entity in tracimcli delete command, i'm not sure about how to make this less tricky.